app/models/concerns/foreman_git_templates/orchestration/tftp.rb

```diff
@@ -29,6 +29,13 @@ def validate_tftp
 
           failure _('No %{kind} template was found for host %{host}. Repository url: %{url}') % { kind: pxe_kind, host: host.name, url: host.params['template_url'] }
         end
+
+        def host_supported?
```

Comment:
Nit: This is included into the Nic model, so this would read `nic.host_supported?` which isn't really descriptive.

```suggestion
        def feasible_for_git_template_rendering?
```

Reply:
I've changed the name 👍